Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># GRUPPE-REPOSITORY\n# TABELL\n| Navn                   | Individuell repository link |\n|------------------------|-----------------------------|\n| Ole Aleksander Paulsen | [Individuell repository](https://github.com/OlePaulsen/IND-REPOSITORY) |\n| Anita Øygard           | [Individuell repository](https://github.com/anitaoygard/ind-repository) |\n| Martine Sinding        | [Individuell repository](https://github.com/MartineSinding/IND-REPOSITORY) |\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 928a47bc9cf034cbeb7c9f521206d53497caf9ab)
+++ b/README.md	(date 1662453136809)
@@ -5,3 +5,4 @@
 | Ole Aleksander Paulsen | [Individuell repository](https://github.com/OlePaulsen/IND-REPOSITORY) |
 | Anita Øygard           | [Individuell repository](https://github.com/anitaoygard/ind-repository) |
 | Martine Sinding        | [Individuell repository](https://github.com/MartineSinding/IND-REPOSITORY) |
+ Ayat Farhat             | [Individuell repository](https://github.com/ayatfarhat/IND-REPOSITORY)
\ No newline at end of file
