Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># GRUPPE-REPOSITORY\n# TABELL\n| Navn | Individuell repository link |\n|--|-----------------------------|\n|Ole Aleksander Paulsen | [Individuell repository](https://github.com/OlePaulsen/IND-REPOSITORY) |\n|Martine Sinding | [Individuell repository](https://github.com/MartineSinding/IND-REPOSITORY) |\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision d4e571d858c949f64f3090c24b4b7030cd965596)
+++ b/README.md	(date 1662549672635)
@@ -4,3 +4,4 @@
 |--|-----------------------------|
 |Ole Aleksander Paulsen | [Individuell repository](https://github.com/OlePaulsen/IND-REPOSITORY) |
 |Martine Sinding | [Individuell repository](https://github.com/MartineSinding/IND-REPOSITORY) |
+x
\ No newline at end of file
